tests/dummy/app/app.js

```diff
@@ -1,12 +1,19 @@
+/* global Prism */
 import Application from '@ember/application';
 import Resolver from 'ember-resolver';
 import loadInitializers from 'ember-load-initializers';
 import config from 'dummy/config/environment';
 
+import { setup } from 'ember-prism';
```

Comment:
each consuming app will have to do this, unless there is a `field-guide` entrypoint we can have folks import and do this setup on behalf of people.